packages/evm/examples/4844.ts

```diff
@@ -0,0 +1,3 @@
+import { Common, Chain, Hardfork } from '@ethereumjs/common'
+
+const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Shanghai, eips: [4844] })
```

Comment:
Shouldn't we point this to `hardfork: Hardfork.Cancun?`
```suggestion
const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Cancun })
```

Reply:
I don't think so.  The idea is to activate an EIP that isn't active on that hardfork and 4844 should be active on Cancun.

Reply:
I see, but then the description in the README should change I think, since now it is:

```
#### Initialization

To run EVM related EIP-4844 functionality you have to active the EIP in the associated `@ethereumjs/common` library:
```

This should then be changed to something like: if you have an EIP that is not active on this hardfork, but want it activated anyways, then use this `eips` param

Reply:
Fair point, have updated the language in the README on this point.

Reply:
At the end no big deal and for sure not a hill to die upon, but I’ve also updated similar examples in a way Jochem suggested.